projects/orquestra-sdk/docs/examples/tests/test_parametrized_workflows.py

```diff
@@ -126,7 +126,9 @@ def test_get_results(change_test_dir, shared_ray_conn, change_db_location):
         # When
         # Call the snippet
         proc = subprocess.run([sys.executable, str(src_file)], capture_output=True)
-
+        assert (
+            False
+        ), f"STDOUT: {proc.stdout.decode()}, \n STDERR: {proc.stderr.decode()}"
```

Comment:
Query: would you mind explaining what this is for?

Reply:
Yeh sorry, python tests on 3.9 do not pass for some reason on CI. Im RN debugging why. Should've taken this PR to draft for that